Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1371,9 +1371,13 @@ extension BrowserViewController: WKNavigationDelegate {
         if let visitType = self.getVisitTypeForTab(tab, navigation: navigation)?.rawValue {
             info["visitType"] = visitType
         }
-        notificationCenter.postNotificationName("LocationChange", object: self, userInfo: info)
 
         if let url = webView.URL {
```

Comment:
Looks good. Cool Swift Tip: Since 1.2 you can do this:

```
if let url = webView.URL where ErrorPageHelper.isErrorPageURL(url) == false {
   ...
}
```